ddl/ddl.go

```diff
@@ -64,7 +64,7 @@ var (
 	// EnableSplitTableRegion is a flag to decide whether to split a new region for
 	// a newly created table. It takes effect only if the Storage supports split
 	// region.
-	EnableSplitTableRegion = false
```

Comment:
It's better to cherry-pick #8719 firstly, rather than cherry-picking 2 commits together.

Reply:
Maybe I can simply change the PR title.